packages/lodestar/test/unit/network/reqresp/encoders/response.test.ts

```diff
@@ -0,0 +1,316 @@
+import chai from "chai";
```

Comment:
Merged responseEncode and responseDecode tests in a single file with common test cases